package.json

```diff
@@ -45,6 +49,8 @@
     "@types/jasmine": "~2.8.8",
     "@types/jasminewd2": "~2.0.3",
     "codelyzer": "4.5.0",
+    "devextreme-cli": "latest",
```

Comment:
Set a specific version, otherwise it could stop working in the feature if the library introduces a change that causes us conflicts